src/uu/cut/src/cut.rs

```diff
@@ -356,28 +368,26 @@ fn cut_files(mut filenames: Vec<String>, mode: Mode) -> i32 {
                 continue;
             }
 
-            if path.metadata().is_err() {
-                show_error!("{}: No such file or directory", filename.maybe_quote());
-                continue;
-            }
+            match File::open(&path).map_err_context(|| filename.maybe_quote().to_string()) {
+                Ok(file) => {
+                    match mode {
+                        Mode::Bytes(ref ranges, ref opts) => cut_bytes(file, ranges, opts)?,
+                        Mode::Characters(ref ranges, ref opts) => cut_bytes(file, ranges, opts)?,
+                        Mode::Fields(ref ranges, ref opts) => cut_fields(file, ranges, opts)?,
+                    };
+                }
```

Comment:
It's probably a rare occurrence, but using the `?` operator will stop the loop, so I would do something like this:
```rust
show_if_err!(File::open(&path)
  .map_err_context(|| filename.maybe_quote().to_string())
  .and_then(|file| {
      match &mode {
          Mode::Bytes(ranges, opts) => cut_bytes(file, ranges, opts),
          Mode::Characters(ranges, opts) => cut_bytes(file, ranges, opts),
          Mode::Fields(ranges, opts) => cut_fields(file, ranges, opts),
      }
  }
));
```

Reply:
Done!